pythonbits/bb.py

```diff
@@ -209,6 +218,31 @@ class VideoSubmission(BbSubmission):
     def _render_guess(self):
         return {k: v for k, v in guessit.guessit(self['path']).items()}
 
+    def _render_confirmed_guess(self):
+        guess = {k: v for k, v in guessit.guessit(self['path']).items()}
+        print('Confirm by pressing enter or type in correct values:')
+
+        guess['title'] = input('Guessed title: %s > ' % (guess['title'],)) or guess['title']
+
+        confirmed_type = None
+        while confirmed_type not in ('movie', 'episode'):
+            user_type = 'tv' if guess['type'] == 'episode' else guess['type']
+            confirmed_type = (input('Guessed type ("movie" or "tv"): %s > ' % (user_type,))
+                              or guess['type'])
+            confirmed_type = 'episode' if confirmed_type == 'tv' else confirmed_type
+        guess['type'] = confirmed_type
+
+        if guess['type'] == 'episode' and 'season' not in guess:
+            while True:
+                try:
+                    guess['season'] = int(input('Season > '))
+                except (ValueError, TypeError):
+                    pass
+                else:
+                    break
+
+        return guess
+
```

Comment:
mhhhh

Reply:
It's possible the "confirmed_guess" stuff can be removed. I'll look into it over the weekend.

Reply:
I'm not too sure if the use case for this is really there, I will check it out tonight.

Reply:
I think I've added this as a quick and dirty fix when guessit was confused/bugged, but I don't remember much else. I'd like to have something like this because I think it's important to not rely on autodetection, but this implementation is probably bad. Ideally we would have a TUI form that is prefilled and can be edited by the user if necessary.

Reply:
OK, I figured it out. TV show packs that don't specify a season or do it badly are detected as "movie" and things break horribly. I've made a few commits that clean this up. The "confirmed_guess" thing is still pretty dirty, but it works and allows easy recovery from otherwise fatal misdetection. Please let's keep it until someone has time to improve.

Reply:
Sorry, but if that is the point, then the fix should be in `VideoSubmission.subcategory()`. This adds a lot of needless complexity. But also if it were implemented there, how often does this realistically happen? And how much time is lost by having to confirm this every time vs. just rerunning it with a manual title? I have removed it in the merged commits. 

If a TV show pack doesn't specify a season you can just run it as e.g. `pythonbits /some/strange/path "breaking bad s01"`

Reply:
`confirmed_request` is only used if the info doesn't make sense, e.g. tv show but no season. Otherwise it has no effect. This happened at least twice to me, and once is enough to make this a bug in my opinion.

But I didn't realize I can put info in the TITLE argument. I thought it was literally the show/movie title. I liked the idea of requesting additional info from the user ad hoc, but I guess this is also a fix.